src/PerformanceUtils.php

```
@@ -0,0 +1,255 @@
+                }
+            }
+        }
+
+        // Include SecurityUtils and bootstrap changes
+        $bootstrapFile = $rootDir . '/includes/bootstrap.php';
+        if (file_exists($bootstrapFile)) {
+            $maxMTime = max($maxMTime, (int) filemtime($bootstrapFile));
+        }
+
+        return $maxMTime;
+    }
```

PHPMD (2.15.0)

[warning] The method getSourceMaxMTime() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)